### PR TITLE
Update tool paths to biotoolsID and introduce a tool annotation - `tool.json` file for each tool

### PR DESCRIPTION
**Pull request by vedran-kasalica**
- Updated paths for each tool to match their biotoolsID (where available)
- Updated some of the CWL files to 1.2. The ideas is to update all of them, but they should be tested before we do that.
- Added a tool annotation - `tool.json` file for each tool to allow reusability of the tools in new domains

The PR will be hard to review (sorry for that), because of the path changes, some files are considered to be deleted instead of moved or renamed.

Should be merger after #70

### CODE REVIEW

File: cwl-tools/comet/comet.cwl [hunk 1]
```diff
@@ -21,7 +21,7 @@ inputs:
       separate: false
     default:
       class: File
-      location: https://raw.githubusercontent.com/Workflomics/tools-and-domains/main/cwl-tools/Comet/comet.params
+      location: https://raw.githubusercontent.com/Workflomics/tools-and-domains/main/cwl-tools/cwl-tools/comet/comet.params
```
Comment (kretep) [inline cwl-tools/comet/comet.cwl:24]: 2x cwl-tools it seems

Comment (vedran-kasalica) [reply cwl-tools/comet/comet.cwl]: Good catch, I just updated it!

**Comment by kretep** [comment]
cwl-tools/protXml2IdList/tool.json seems to be empty, but I guess it shouldn't be?

**Comment by vedran-kasalica** [comment]
> cwl-tools/protXml2IdList/tool.json seems to be empty, but I guess it shouldn't be?

Indeed, I added the annotations with the latest comit.

**Pull request merged**